packages/components/link/src/link.tsx

```diff
@@ -38,6 +43,13 @@ type TLinkProps = {
    * Color of the link
    */
   tone?: 'primary' | 'inverted';
+
+  /**
+   * Handler when the link is clicked.
+   */
+  onClick?: (
```

Comment:
Wasn't this part of another PR? #2323

Reply:
Yes but the changes then shouldn't be visible in your PR.

I would suggest to rebase the branch instead of merging it.

Reply:
Ah, yes you're right. 😅